flang/lib/Lower/ConvertExpr.cpp

```diff
@@ -355,6 +370,178 @@ static bool allSymbolsInExprPresentInMap(const Fortran::lower::SomeExpr &expr,
   return true;
 }
 
+/// Generate a load of a value from an address. Beware that this will lose
+/// any dynamic type information for polymorphic entities.
```

Comment:
How can a value of `!fir.ref<none>` be loaded? That doesn't make sense.

Reply:
I think the trouble with this for me is that unlimited polymorphic entities _are_ polymorphic entities. Just more so. So the terminology drops something on the floor.

Seems like an easy fix to just say "but not unlimited polymorphic entities" in the comments.

Thanks for clarifying.